.changeset/sweet-spies-drum.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-api": minor
+---
+
+Allow setting individual permissions per operation as this can be required if some users may not be allowed to update an entity. This is a additional option so no changes required.
```

Comment:
```suggestion
API Generator: Allow setting individual permissions per operation

This can be necessary if some users aren't allowed to perform certain operations, e.g., updating an entity.
```

The changeset should typically have a short title and optionally additional description in a new line if necessary.

Reply:
thank you for this info and your fix, I used the changeset cli command and there was no hint how this should be formatted.